geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegion.java

```diff
@@ -738,7 +738,7 @@ public boolean isPdxTypesRegion() {
   }
 
   boolean isGatewaySenderEnabled() {
```

Comment:
can we remove this method and the code that calls it?

Reply:
Will check...

Reply:
Oh yeah, I meant to go back and look at this. It isn't clear why this was still in the code. There are lots of checks but there are no writes to the private variable in this class. I guess it was a refactoring that never clean it up. I want to do a bit more digging to make sure this isn't a hidden bug.

Reply:
Ok, looks like you removed it in f7b2714c14a6eeab4e810a85f65d650483f2ce6f. Does it ring a bell?

Reply:
Have a look at 26056ef, does this look correct to have all this gateway sender stuff stripped out of LocalRegion?

Reply:
👍that looks good